### ISSUE TITLE
chat messages and state changed events should be in aria live region

### ISSUE THREAD

**Issue by albertotirla** [owncast/owncast#3335]
## type

bug

## owncast version

0.1.1

## description

a lively stream needs live feedback. This applies both in the case of users animatedly talking to one another or with the host, as well as if the host needs a screenreader to successfully use the computer.

Getting events as soon as possible, in this case, is important because it's related to live discussion.

The way this is currently done is by manually going in the chat buffer and reading messages. For bigger chat, this is a problem, because catching up to messages is difficult with the current scrolling system, plus it's a big stream after all.

Even as a user it's difficult, because while the placement of their cursor, aka unread messages, isn't as important as for the owner/moderator, not getting a reply to a message because one didn't check the message buffer often enough isn't cool

## steps to reproduce

* turn on a screenreader. Good examples are narrator for windows, talkback for android, voiceover for apple devices
* open an owncast stream
* either generate traffic yourself, or go to a stream which already has a high activity
* observe, screenreader doesn't read some things

## actual behaviour

chat isn't read automatically by the screenreader when it comes in. Same about people joining, changing their nicknames, authorising with fedi, leaving, etc

## expected behaviour

chat is read automatically, perhaps with a sound per message if the window of the browser isn't focused and screenreader can't read the aria region because of that, see mastodon. Also, optionally have other events in the aria region, such as person joins, changes name, leaves, etc

## remarcs

this is the reverse problem to #3334, where two much information was dumped in the live region. If both of these are solved, owncast would suddanly become much more accessible

**Comment by yerimsong** [comment]
We're a group of 3 CMU students and we'd like to contribute to an open-source project! Is this issue still available?

**Comment by gabek** [comment]
It is, thanks!

**Comment by riybha216** [comment]
Hi @gabek, we've created a PR that resolves this issue. @melghali and I are both part of @yerimsong's group above, and we would love to receive feedback on the changes made!
